Documentation/data-collection.md

```diff
@@ -604,6 +604,16 @@ data:
     # Number of jaeger instances used certain agent strategy
     - '{__name__="jaeger_operator_instances_agent_strategies"}'
     #
+    # owners: (@tracing-team)
+    #
+    # Number of jaeger instances used autoprovisioning feature
+    - '{__name__="jaeger_operator_instances_autoprovisioning"}'
```

Comment:
to reduce the metric cardinality, it would be good to have recording rules aggregating away unnecessary labels like instance, pod, ...

See https://github.com/openshift/cluster-monitoring-operator/pull/1536 for an example.